### ISSUE TITLE
No sandbox installation on node 15.x

### ISSUE THREAD

**Issue by davidethebolo** [akamai/cli-sandbox#70]
It is no possible to install sandbox:
➜  akamai install sandbox 
Attempting to fetch command from https://github.com/akamai/cli-sandbox.git...
Attempting to fetch command from https://github.com/akamai/cli-sandbox.git...... [OK]

Installing...
TRACE[0001] Node.js binary found: /usr/local/bin/node    
TRACE[0001] /usr/local/bin/node -v: v15.3.0              
INFO[0001] package.json found, running npm package manager 
DEBUG[0004] Unable execute package manager (/usr/local/bin/npm install):  
DEBUG[0004] npm ERR! code ENOTFOUND                      
DEBUG[0004] npm ERR! syscall getaddrinfo                 
DEBUG[0004] npm ERR! errno ENOTFOUND                     
DEBUG[0004] npm ERR! network request to https://repos.akamai.com:443/api/npm/npmjs-repos/xtend/-/xtend-4.0.1.tgz failed, reason: getaddrinfo ENOTFOUND repos.akamai.com 

Do you know what it could be?

**Comment by krzyk** [comment]
@davidethebolo  Could you check if you don't have any overrides in .npmrc?
The easiet way would be by doing:
```
npm config list
```

If you have `registry = ` please comment out this line and try again `akamai install sandbox`

**Comment by davidethebolo** [comment]
I used an old version of node (v14.15.1 instead of v15.xx) to install it

**Comment by herzykj** [comment]
sandbox cli should work correctly on node 8.x and higher.

**Comment by krzyk** [comment]
@davidethebolo the culpirt is here (from your logs):

```
DEBUG[0004] npm ERR! network request to https://repos.akamai.com:443/api/npm/npmjs-repos/xtend/-/xtend-4.0.1.tgz failed, 
```

Could you double check if your `.npmrc` doesn't have that repo?
Easiet way is to use `npm config list`

**Comment by andrewmerrell** [comment]
Similar issue, it wont install on `v15.1.0` but works fine on `v12.7.0`, I didn't change any npm settings.

**Comment by krzyk** [comment]
@andrewmerrell  @davidethebolo Indeed when I setup CI tests with node 15.x it fails (https://github.com/akamai/cli-sandbox/pull/77/checks?check_run_id=1655409010)

I will work on it to get it fixed.

**Comment by akamagorecki** [comment]
This was fixed in 1.3.0 version release. Please reopen if you encounter any issue now.
